docs/README.md

```diff
@@ -152,23 +152,25 @@ forc index check
 +--------+------------------------+---------------------------------------------------------+
 | Status |       Component        |                         Details                         |
 +--------+------------------------+---------------------------------------------------------+
-|   ✅   | fuel-indexer binary    |  /Users/me/.fuelup/bin/fuel-indexer                     |
+|   ⛔️   | fuel-indexer binary    |  Can't locate fuel-indexer.                             |
 +--------+------------------------+---------------------------------------------------------+
-|   ⛔️   | fuel-indexer service   |  Failed to detect service at Port(29987).               |
+|   ✅   | fuel-indexer service   |  Local service found: PID(63967) | Port(29987).         |
 +--------+------------------------+---------------------------------------------------------+
 |   ✅   | psql                   |  /usr/local/bin/psql                                    |
 +--------+------------------------+---------------------------------------------------------+
-|   ✅   | fuel-core              |  /Users/me/.fuelup/bin/fuel-core                        |
+|   ✅   | fuel-core              |  /Users/rashad/.cargo/bin/fuel-core                     |
 +--------+------------------------+---------------------------------------------------------+
 |   ✅   | docker                 |  /usr/local/bin/docker                                  |
 +--------+------------------------+---------------------------------------------------------+
-|   ✅   | fuelup                 |  /Users/me/.fuelup/bin/fuelup                           |
+|   ⛔️   | fuelup                 |  Can't locate fuelup.                                   |
 +--------+------------------------+---------------------------------------------------------+
-|   ✅   | wasm-snip              |  /Users/me/.cargo/bin/wasm-snip                         |
+|   ✅   | wasm-snip              |  /Users/rashad/.cargo/bin/wasm-snip                     |
 +--------+------------------------+---------------------------------------------------------+
-|   ✅   | forc-postgres          |  /Users/me/.fuelup/bin/fuelup                           |
+|   ⛔️   | forc-postgres          |  Can't locate fuelup.                                   |
 +--------+------------------------+---------------------------------------------------------+
-|   ✅   | rustc                  |  /Users/me/.cargo/bin/rustc                             |
+|   ✅   | rustc                  |  /Users/rashad/.cargo/bin/rustc                         |
++--------+------------------------+---------------------------------------------------------+
+|   ✅   | forc-wallet            |  /Users/rashad/.cargo/bin/forc-wallet                   |
 +--------+------------------------+---------------------------------------------------------+
```

Comment:
Love the changes here. We should probably decide for good whether we care about having one of our names in the README example outputs. 😅

Reply:
I thought this was intentional haha